src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/TypeLoaderTypeSystemContext.cs

```diff
@@ -100,6 +100,11 @@ protected override RuntimeInterfacesAlgorithm GetRuntimeInterfacesAlgorithmForDe
             }
         }
 
+        protected sealed internal override bool IsIDynamicInterfaceCastableInterface(DefType type)
+        {
+            throw new NotImplementedException();
```

Comment:
Is this only needed for universal generics?

Reply:
We'll _probably_ only need to implement this for the "load type from ECMA-335 metadata" case. We can just copy this bit over from the template for both univeral and non-universal generics. We only need to provide an implementation because this method is abstract and we don't inherit the regular implementation because the regular implementation assumes we have full metadata for everything and can do name checks.

The implementation will likely be similar to this:

https://github.com/dotnet/corert/blob/4a57effbd165a07593282b9d588f410457239637/src/System.Private.TypeLoader/src/Internal/Runtime/TypeLoader/TypeLoaderTypeSystemContext.cs#L281-L295